### ISSUE TITLE
Linux HIDPI support

### ISSUE THREAD

**Issue by gluon-bot** [gluonhq/scenebuilder#66]
Originally reported by: **Karel Van Hecke (Bitbucket: [karelvanhecke](https://bitbucket.org/karelvanhecke), GitHub: [karelvanhecke](https://github.com/karelvanhecke))**

----------------------------------------

The application is completely unusable with a high resolution display. Since it already has HiDPI scaling on Windows, would it be possible to add HiDPI scaling on linux?

----------------------------------------
- Bitbucket: https://bitbucket.org/gluon-oss/scenebuilder/issue/66

**Comment by amalic** [comment]
Any news when this will be fixed?

**Comment by abhinayagarwal** [comment]
@amalic The oracle ticket linked by you has been fixed in OpenJFX 17. Can you still reproduce the issue while running Scene Builder with JavaFX 17-ea builds?

**Comment by amalic** [comment]
Yes. I think it has to do with fractional scaling. GDK_SCALE is per default set to 1.25. My guess is  that JavaFX converts it to an integer?

**Comment by abhinayagarwal** [comment]
Strange, I can't reproduce the issue when using 125% scaling on Ubuntu 20.04.2 LTS and running from master on JavaFX 17-ea+14.

**Comment by amalic** [comment]
Really strange. Maybe it's a driver or version thing (NVidia CUDA drivers, Open JDK 16, ...). Or my second screen which is rotated by 90°>

**Comment by abhinayagarwal** [comment]
How are you running Scene Builder?

**Comment by amalic** [comment]
I installed the Debian package via `apt install`. Running it via provided Ubuntu desktop launcher, except for my scaling fix.

**Comment by abhinayagarwal** [comment]
Hi @amalic ,

Debian package comes with JavaFX version 16. Can you run Scene Builder JavaFX 17-ea+14 to check if the issue is resolved?

Just follow these steps:

1. Clone scenebuillder repository
2. Update [JavaFX version in root pom.xml](https://github.com/gluonhq/scenebuilder/blob/master/pom.xml#L19) to "17-ea+14"
3. run `mvn install && mvn javafx:run -f app`
